internal/commands/push.go

```diff
@@ -102,6 +105,9 @@ func prepareToImport(now Now, tm *model.ThingModel, raw []byte, optPath string)
 	if !generatedId.Equals(idFromFile) {
 		finalId = generatedId
 	}
+	if len(finalId.Name) > maxNameLength {
+		return nil, model.TMID{}, fmt.Errorf("TM name too long (max %d allowed): %s", maxNameLength, finalId.Name)
```

Comment:
We should define that as a sentinel error, that we can map it in the http API to a 400 bad request error.
At the moment it results in a common 500 error.

Reply:
you're absolutely right. fixed now